src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,37 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({
+  movie: { imgUrl, title, description, imdbUrl },
+}) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img data-cy="MovieImage" src={imgUrl} alt={title} />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="images/imdb-logo.jpeg" alt="imdb" />
```

Comment:
More correctly to provide different alt for each movie card. Use movie title for exmpl